index.d.ts

```diff
@@ -0,0 +1,28 @@
+export default class Kareem {
```

Comment:
i dont know if it would be necessary, but maybe add a `declare module 'Kareem'`

Reply:
Fixed in 3bfa755